packages/service/README.md

```diff
@@ -334,15 +342,76 @@ interface RgbppApiCkbTransactionHash {
   txhash: string;
 }
 
+interface RgbppApiTransactionStateParams {
+  with_data?: boolean;
+}
+
 interface RgbppApiTransactionState {
   state: RgbppTransactionState;
+  attempts: number;
+  failedReason?: string;
+  data?: {
+    txid: string;
+    ckbVirtualResult: {
+      ckbRawTx: CKBComponents.RawTransaction;
+      needPaymasterCell: boolean;
+      sumInputsCapacity: string;
+      commitment: string;
+    };
+  };
+}
+
+interface RgbppCell extends Cell {
+  typeHash?: Hash;
 }
 
 interface RgbppApiAssetsByAddressParams {
   type_script?: string;
   no_cache?: boolean;
 }
 
+interface RgbppApiBalanceByAddressParams {
+  type_script?: string;
+  no_cache?: boolean;
+}
+interface RgbppApiBalance {
+  address: string;
+  xudt: RgbppApiXudtBalance[];
+}
+interface RgbppApiXudtBalance {
+  name: string;
+  decimal: number;
+  symbol: string;
+  total_amount: string;
+  available_amount: string;
+  pending_amount: string;
+  type_hash: string;
+  type_script: Script;
+}
+
+interface RgbppApiActivityByAddressParams {
+  rgbpp_only?: boolean;
+  type_script?: string;
+  after_btc_txid?: string;
+}
+interface RgbppApiActivity {
+  address: string;
+  cursor: string;
+  txs: {
+    btcTx: BtcApiTransaction;
+    isRgbpp: boolean;
+    isomorphicTx?: {
+      ckbRawTx?: CKBComponents.RawTransaction;
+      ckbTx?: CKBComponents.Transaction;
+      inputs?: CKBComponents.CellOutput[];
+      outputs?: CKBComponents.CellOutput[];
```

Comment:
I cannot find the `inputs` and  `outputs` from the [API document](https://www.notion.so/Assets-API-Activity-JoyID-d4a326cf6ddc47da806c72f5aa5852d9) and we can get them from the `ckbTx`.

I don't understand why the response still includes `inputs` and `outputs`.

Reply:
Understood with https://github.com/ckb-cell/btc-assets-api/pull/182#discussion_r1679202822